services/galley/test/integration/API/Teams.hs

```diff
@@ -888,20 +888,21 @@ postCryptoBroadcastMessageProto g b c a = do
         -- Alice should not get her own broadcast
         WS.assertNoEvent timeout ws
 
-postCryptoBroadcastMessageNoTeam :: Galley -> Brig -> Cannon -> Maybe Aws.Env -> Http ()
-postCryptoBroadcastMessageNoTeam g b _ _ = do
+postCryptoBroadcastMessageNoTeam :: Galley -> Brig -> Cannon -> Maybe Aws.Env -> TestSetup -> Http ()
+postCryptoBroadcastMessageNoTeam g b _ _ _ = do
     (alice, ac) <- randomUserWithClient b (someLastPrekeys !! 0)
     (bob,   bc) <- randomUserWithClient b (someLastPrekeys !! 1)
     connectUsers b alice (list1 bob [])
     let msg = [(bob, bc, "ciphertext1")]
     Util.postOtrBroadcastMessage id g alice ac msg !!! const 404 === statusCode
 
-postCryptoBroadcastMessage100 :: Galley -> Brig -> Cannon -> Maybe Aws.Env -> Http ()
-postCryptoBroadcastMessage100 g b c a = do
+postCryptoBroadcastMessageMaxConns :: Galley -> Brig -> Cannon -> Maybe Aws.Env -> TestSetup -> Http ()
+postCryptoBroadcastMessageMaxConns g b c a s = do
+    let maxConns = fromIntegral (maxConvTeamSize s)
```

Comment:
At the moment, the default for maxConvTeamSize is 128, while the default for the maximum number of connections is 1000. While this test could benefit from a configurable amount of connected users, I think we shouldn't mix these two constants here.

Reply:
Good point. However, since `galley` does not know what this number is (and we need to since creating more connections will cause the test to fail), I suggest we create as many connections as possible (i.e., until `brig` returns us a `403`) or we reach 100 users, whichever comes first.
https://github.com/wireapp/wire-server/pull/207/commits/34446804188851a3599560c02dd7d40a5b4ae998